data_steward/cdr_cleaner/cleaning_rules/drop_duplicate_states.py

```diff
@@ -0,0 +1,88 @@
+"""
+ Currently there are some individuals with multiple state records from consent.
```

Comment:
Needs a summary line followed by a blank line for Pep-3/Pep-257 documentation standards.  Like:

```
Drop duplicate states from participant records.

Currently there are some  individuals...
```